features/step_definitions/common_steps.rb

```diff
@@ -266,7 +266,11 @@
   begin
     banner_message = page.find(:css, ".dm-banner, .banner-#{status}-without-action", wait: false)
   rescue Capybara::ElementNotFound => e
-    banner_message = page.find(:css, ".banner-#{status}-with-action", wait: false)
+    begin
+      banner_message = page.find(:css, ".banner-#{status}-with-action", wait: false)
```

Comment:
Is the old-style banner still being used?

Reply:
@bjgill it appears it is still being used in other scenarios.